packages/p2p-v2/types.ts

```diff
@@ -13,9 +13,18 @@ declare global {
 }
 
 export type TAdvertiserPaymentMethods = ReturnType<typeof p2p.advertiserPaymentMethods.useGet>['data'];
-
+export type TPaymentMethods = ReturnType<typeof p2p.paymentMethods.useGet>['data'];
+export type TAccumulatedPaymentMethods = Record<
+    string,
+    NonNullable<TAdvertiserPaymentMethods | TPaymentMethods>[number]
+>;
+export type TPaymentMethod = NonNullable<TAdvertiserPaymentMethods>[number] | NonNullable<TPaymentMethods>[number];
+type TPaymentMethodDisplayName = NonNullable<
+    ReturnType<typeof p2p.paymentMethods.useGet>['data']
+>[number]['display_name'];
 export type TSelectedPaymentMethod = Partial<{
-    displayName: NonNullable<ReturnType<typeof p2p.paymentMethods.useGet>['data']>[number]['display_name'];
+    displayName: TPaymentMethodDisplayName;
+    display_name: TPaymentMethodDisplayName;
```

Comment:
I don't think this is ideal @nada-deriv,  the transformation should be made at consumer level. Having two similar properties that represent the same thing might lead to bugs, because the two might get out of sync
```suggestion
    displayName: TPaymentMethodDisplayName;
    display_name: TPaymentMethodDisplayName;
```

Reply:
had added it for handling something related to add payment method in sell form, removed it for now, since that part is not implemented. will handle it different later.